src/ORM/Filters/ExactMatchFilter.php

```diff
@@ -205,4 +221,38 @@ public function isEmpty()
     {
         return $this->getValue() === [] || $this->getValue() === null || $this->getValue() === '';
     }
+
+    /**
+     * Determine if we can use placeholders for the given column and values
+     * Current rules are to use placeholders for all values, unless:
+     * - The column is a DBPrimaryKey or DBForeignKey
+     * - The values being filtered are all either integers or valid integer strings
+     * - Using placeholders for integer ids has been configured off, which is the default config value
```

Comment:
```suggestion
     * - Using placeholders for integer ids has been configured off
```
We could arguably change the default config later - we don't want this comment to become stale

Reply:
Updated